.vscode/settings.json

```diff
@@ -7,5 +7,6 @@
         "out": true // set this to false to include "out" folder in search results
```

Comment:
axe comments? json doesn't support comments far as I know 🤔

Reply:
actually those were created by vscode and are valid for the setting file :)

Reply:
TIL... lol, thanks for the heads up